frontend/src/components/ViewActivities/index.js

```diff
@@ -1,16 +1,91 @@
 import React from 'react';
+import app from '../Firebase';
+import { getUserUid } from '../AuthUtils';
+import ActivityList from './activitylist.js';
+
+import * as DB from '../../constants/database.js';
```

Comment:
What is the general style used for import spacing? I don't understand the difference between line 4 and 6 that warranted having the space between them.

Reply:
I think we've settled on:
```
React stuff

Firestore, Bootstrap, other libraries

Local imports
```

I've updated the spacing to reflect this.